core/include/panic.h

```diff
@@ -74,6 +74,14 @@ typedef enum {
  * */
 NORETURN void core_panic(core_panic_t crash_code, const char *message);
 
+/**
+ * @brief architecture dependent handling of an panic case
+ *
+ * This function gives the CPU the possibility to execute architecture
+ * dependent code in case of an severe error.
```

Comment:
s/an/a/

Reply:
Boths typos were copy&pasted...

Reply:
Yes, I know, but it can be the good time to fix them ;)

Reply:
There you are. Still I thought we outlawed "while you're at it" review comments... ;)